common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -83,6 +83,7 @@
 from .caching_descriptor_system import CachingDescriptorSystem
 from xmodule.modulestore.split_mongo.mongo_connection import MongoConnection, DuplicateKeyError
 from xmodule.modulestore.split_mongo import BlockKey, CourseEnvelope
+from xmodule.modulestore.mongo.base import _DETACHED_CATEGORIES
```

Comment:
this feels weird, for split to import from `mongo.base`. Can they both just import from `xmodule.modulestore`? (You can put `DETACHED_CATEGORIES` into `xmodule/modulestore/__init__.py`. I'd also remove the leading underscore, since it will no longer be private.